com.github.weclaw1.ScoreTracker.yaml

```diff
@@ -0,0 +1,33 @@
+app-id: com.github.weclaw1.ScoreTracker
```

Comment:
as per the documentation, theis app-id should start with `io.github.`

Reply:
Fixed, I also released a new version with updated app id